force-app/main/domain/Households.cls

```diff
@@ -29,10 +29,12 @@
 */
 /**
 * @author Salesforce.org
-* @date 2021
-* @description Contact repository responsible for retrieval and storing of Contacts for specified Accounts
+* @description Domain class for Household accounts.
+ * THIS CLASS SHOULD NOT CONTAIN ANY PUBLIC CONSTRUCTORS. To retrieve an instance of this domain, use the public static
+ * newInstance() method to retrieve the domain instance from the domain factory. Logic is used in the inner
+ * Constructor class to ensure only household records are contained in the instanced domain collection.
 */
-public inherited sharing class Households extends fflib_SObjects2 {
+public with sharing class Households extends fflib_SObjects2 implements IHouseholds {
```

Comment:
This must be `inherited sharing`. Best practice is for non-UI controllers to use `inherited sharing`. Utility classes that require special access can use `without sharing`. Only UI (i.e., only used directly by a customer UI) should use `with sharing`.